src/main/resources/static/js/MainWindow.js

```diff
@@ -1,670 +1,756 @@
+			loadThing(result.thing.frostId, streams, url);
+		},
+		error : function(e) {
+			addToLog(e.responseText);
+		}
+	});
 }
 
 function loadStreamConfig(stream, div) {
 	var url = document.getElementById("serverurlbox").innerText;
-    $.ajax({
-        type: "GET",
-        url: "datastream/single",
-        data: {
-            id: parseInt(stream.dsID),
-            isMulti: stream.multiStream,
-            url: url
-        },
-        success: function (result) {
-            div.find("select").val(result.name + " (" + result.frostId + ")").trigger("change");
-            div.find("select option:selected").attr("data-value", JSON.stringify(result));
-            loadStream(result, div);
-            loadStreamCol(stream.observations, div);
-        },
-        error: function (e) {
-            addToLog(e.responseText);
-        }
-    });
+	$.ajax({
+		type : "GET",
+		url : "datastream/single",
+		data : {
+			id : parseInt(stream.dsID, 10),
+			isMulti : stream.multiStream,
+			url : url
+		},
+		success : function(result) {
+			div.find("select").val(result.name + " (" + result.frostId + ")")
+					.trigger("change");
+			div.find("select option:selected").attr("data-value",
+					JSON.stringify(result));
+			loadStream(result, div);
+			loadStreamCol(stream.observations, div);
+		},
+		error : function(e) {
+			addToLog(e.responseText);
+		}
+	});
 }
 
 function fillStreams(streams) {
-    var child;
-    for (var i = 0; i < streams.length; i++) {
-        addDatastream();
-        child = $("#datastreams > div").last();
-        loadStreamConfig(streams[i], child);
-    }
+	var child;
+	for (var i = 0; i < streams.length; i++) {
+		addDatastream();
+		child = $("#datastreams > div").last();
+		loadStreamConfig(streams[i], child);
+	}
 
 }
 
 /**
  * clears all inputs for the configuration
  */
 function resetConfig() {
-    document.getElementById("delimiter").value = ";";
-    document.getElementById("headerlines").value = "0";
-    $("#things").val(null).trigger("change");
-    $("#selecttime").val(null).trigger("change");
-    $("#timeTable").find("tbody").find("tr").each(function () {
-        this.remove();
-    });
-    addRow($("#timeTable"), "2", "20");
-    document.getElementById("datastreams").innerHTML = "";
-    $("#configs").val(null).trigger("change");
-    resetMapping();
-
-    optimzeforsource(); //disables the delimiter and time format if the source file is an excel file
-    showstep1();
+	document.getElementById("delimiter").value = ";";
+	document.getElementById("headerlines").value = "0";
+	$("#things").val(null).trigger("change");
+	$("#selecttime").val(null).trigger("change");
+	$("#timeTable").find("tbody").find("tr").each(function() {
+		this.remove();
+	});
+	addRow($("#timeTable"), "2", "20");
+	document.getElementById("datastreams").innerHTML = "";
+	$("#configs").val(null).trigger("change");
+	resetMapping();
+
+	optimzeforsource(); // disables the delimiter and time format if the source
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['optimzeforsource' is not defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)